docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,88 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there were three configuration settings for how Mobile UCRs were restored and sent to devices:
+
+- Version 1.0: Bundled all reports into a single fixture during the restore process.
+- Version 1.5: Sent both V1 and V2 restore formats.
+- Version 2.0: Split UCRs into multiple fixtures within the restore.
+
+Deprecation of Mobile UCR V1 and V1.5
+-------------------------------------
+Dimagi deprecated both V1 and V1.5 to address performance challenges encountered with these versions.
```

Comment:
nit: We should avoid saying "Dimagi deprecated" and instead keep it simple as "Both V1 and V1.5 ..."